### ISSUE TITLE
KB-Modified Main Menu

### ISSUE THREAD

**Issue by iBuyMountainDew** [kami-blue/client#775]

**Comment by dominikaaaa** [comment]
Can u fill out description ktnx

**Comment by iBuyMountainDew** [comment]
ez

**Comment by dominikaaaa** [comment]
what does this entail??? what would be changed / added etc

**Comment by dominikaaaa** [comment]
anybody looking at this issue legitimately has now idea what it's for smh

**Comment by iBuyMountainDew** [comment]
This will likely include a KAMI Blue logo placed in the main menu as well as additions to the text in the bottom left and right hand corners related to KAMI Blue.

**Comment by dominikaaaa** [comment]
I have no idea why that would be useful honestly, unless you mean version number and custom status (contributor, donator, all the code for this is already written), with would be neat. 

Maybe have something to access Kami blue's GUI? And a GUI way to add friends, and of course with an alt manager planned (#12) you'd need a button for that as well.

**Comment by dominikaaaa** [comment]
About the splash texts, instead we could have a MOTD like impact does (instead of choosing a random one, it downloads a unique one from GitHub whenever you change it), which I think would be better because you can change it to something funny if something happened (ie Trump telling people to drink bleach, NK's president dying and being resurrected)

**Comment by sourTaste000** [comment]
> Customised main menu, one tailored to KAMI Blue, like in Impact.
> 
> * [ ]  ability to toggle all changes back to Vanilla mc
> * [ ]  logo
> * [ ]  open KAMI GUI button
> * [ ]  version num / nightly hash
> * [ ]  special status (contributor, donator, see [setCustomIcons()](https://github.com/kami-blue/client/blob/c5a933a/src/main/java/me/zeroeightsix/kami/DiscordPresence.java#L84) for this)
> * [ ]  alt manager, when implemented (#12)
> * [ ]  friends gui manager
> * [ ]  custom splash text (ideas [here](https://discordapp.com/channels/573954110454366214/634010583527587840/706562156690276482))

[expanded version?](https://discordapp.com/channels/573954110454366214/634010583527587840/706564911266005063]

**Comment by dominikaaaa** [comment]
What? No, that's literally linked in the issue body.

**Comment by sourTaste000** [comment]
he linked the old one

**Comment by dominikaaaa** [comment]
No, I didn't. I deleted the second message on discord as apparently sending the same message twice causes confusion.

**Comment by sourTaste000** [comment]
ok

**Comment by dominikaaaa** [comment]
Splashes: https://github.com/lumien231/Custom-Main-Menu (MIT) licensed
